crates/bevy_ui/src/render/mod.rs

```diff
@@ -666,6 +668,9 @@ pub fn extract_default_ui_camera_view(
             commands
                 .get_or_spawn(entity)
                 .insert(DefaultCameraView(default_camera_view));
+            if let Some(ui_antialias) = ui_antialias {
+                commands.get_or_spawn(entity).insert(*ui_antialias);
+            }
```

Comment:
Shouldn't need to call `get_or_spawn` twice here?

```suggestion
            let entity_commands = commands
                .get_or_spawn(entity)
                .insert(DefaultCameraView(default_camera_view));
            if let Some(ui_antialias) = ui_antialias {
                entity_commands.get_or_spawn(entity).insert(*ui_antialias);
            }
```

Reply:
Whoops, missed that!